src/routes/user.routes.js

```diff
@@ -1,16 +1,13 @@
 const { Router } = require('express')
 const api = Router()
 const userController = require('../controller/user.controller')
-const verifyEmptyFields = require('../middlewares/verifyEmptyFields')
-const verifyDuplicatesEmails = require('../middlewares/verifyDuplicatesEmails')
-const verifyRole = require('../middlewares/verifyRole')
-const verifyId = require('../middlewares/verifyId')
+const middlewares = require('../middlewares/index')
```

Comment:
## observations regarding this pull request 
### nodejs looks for indexes by default in a directory, so it is not necessary to indicate the index when importing a module 
## current code in this pull reques
```js
const middleware = require('../middlewares/index')
```
## you can use this snippet code instead of the above example
```js
const middleware = require('../middlewares')
```
### This does practically the same thing, only it is good Javascript practice